napari/_qt/widgets/qt_viewer_status_bar.py

```diff
@@ -15,7 +22,49 @@ class ViewerStatusBar(QStatusBar):
     def __init__(self, parent: '_QtMainWindow') -> None:
         super().__init__(parent=parent)
 
-        self.showMessage(trans._('Ready'))
+        main_widget = QWidget()
+
+        layout = QHBoxLayout()
+
+        self._status = QLabel('Ready')
+        self._status.setMinimumSize(0, 16)
+        self._status.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Maximum)
+        self._layer_base = QElidingLabel(trans._(''))
+        self._layer_base.setElideMode(Qt.TextElideMode.ElideMiddle)
+        self._layer_base.setMinimumSize(120, 16)
```

Comment:
@ppwadhwa is there a reason we're setting a fixed minimum here? It would be nice if we could make the text as tight as possible by allowing these widgets to shrink. This looks great for longer names, but for short ones it just looks like so much space...